sdk/monitor/query/azmetrics/README.md

```diff
@@ -1,8 +1,8 @@
 # Azure Monitor Query Metrics client module for Go
 
-The Azure Monitor Query Metrics client module is used to execute read-only queries against [Azure Monitor][azure_monitor_overview]'s metrics platform.
-
-[Metrics][metrics_overview] collects numeric data from monitored resources into a time series database. Metrics are numerical values that are collected at regular intervals and describe some aspect of a system at a particular time. Metrics are lightweight and capable of supporting near real-time scenarios, making them particularly useful for alerting and fast detection of issues.
+* Query metrics (this module)- execute read-only queries against [Azure Monitor Metrics][metrics_overview]
```

Comment:
🤔 this is probably a job for colon
```suggestion
* Query metrics (this module): execute read-only queries against [Azure Monitor Metrics][metrics_overview]
```

Reply:
fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)